webxrlayers-1.bs

```diff
@@ -811,11 +863,44 @@ dictionary XRLayerInit {
 The <dfn dict-member for="XRLayerInit">space</dfn> attribute defines the spatial relationship with the user’s physical environment.
 
 The <dfn dict-member for="XRLayerInit">colorFormat</dfn> attribute defines the data type of the color texture data.
-The default value is {{RGBA}}.
+
+This is the <dfn>list of color formats for non-projection layers</dfn> that the [=XR Compositor=] MUST support:
+ - {{RGBA}}
+ - {{RGB}}
+ - {{SRGB_EXT}} for contexts with the '{{EXT_sRGB}}' [=extension=] enabled
+ - {{SRGB_ALPHA_EXT}} for contexts with the '{{EXT_sRGB}}' [=extension=] enabled
+
+For WebGL2 contexts these additional formats are supported:
+ - {{RGBA8}}
+ - {{RGB8}}
+ - {{SRGB8}}
+ - {{SRGB8_ALPHA8}}
+
+For contexts with the '{{WEBGL_compressed_texture_etc}}' [=extension=] enabled these additional formats are supported:
+ - {{COMPRESSED_RGB8_ETC2}}
+ - {{COMPRESSED_RGB8_PUNCHTHROUGH_ALPHA1_ETC2}}
+ - {{COMPRESSED_RGBA8_ETC2_EAC}}
+ - {{COMPRESSED_SRGB8_ETC2}}
+ - {{COMPRESSED_SRGB8_PUNCHTHROUGH_ALPHA1_ETC2}}
+ - {{COMPRESSED_SRGB8_ALPHA8_ETC2_EAC}}
+
+For contexts with the '{{WEBGL_compressed_texture_astc}}' [=extension=] enabled all the formats of that [=extension=] are supported.
```

Comment:
Is there a reason why only these two compressed texture extensions are called out? I would be surprised, for example, if S3TC wasn't supported on desktop systems.

Reply:
We wanted to create lists of formats that were supported by all systems.  S3TC is not supported on Qualcomm.

Maybe we should revisit and state: "if the extension can be enabled, then the following formats are supported..."

Reply:
That sounds like a good approach, though I'd say that the current verbiage works fine for the scenario where the extension isn't available as well, since you obviously can't enable it in that case.

Reply:
@thetuvix Do you have an opinion? Should we add formats that everyone supports or should we let it depend on the extension + the formats of that extension that make sense (ie no single component or depth for the swapchain)